site/docs/guide/getting-started.md

```diff
@@ -150,6 +150,89 @@ in your terminal before you execute `node bot.js`.
 This makes it easier to debug your bot.
 :::
 
+## Getting Started on Bun
+
+> This guide assumes that you have [Bun](https://bun.sh) installed.
+> If you don't know what these things are, check out our [Introduction](./introduction)!
+
+Create a new project and install the `grammy` package.
+Do this by opening a terminal and typing:
+
+```sh
+# Create a new directory and change into it.
+mkdir my-bot
+cd my-bot
+
+# Run bun init to scaffold a new project, for this tutorial, just press enter to accept the default answer for each prompt.
+bun init
+
+# Install grammY.
+bun add grammy
+```
+
+Create a new empty text file, e.g. called `bot.ts`.
```

Comment:
Aren't bun automatically creating index.ts if we `just press enter to accept the default answer for each prompt.`